common/lib/api-client/middleware/request-include.js

```diff
@@ -1,8 +1,12 @@
 const { get } = require('lodash')
 
 module.exports = {
-  name: 'req-include',
+  name: 'request-include',
   req: function req(payload = {}) {
+    if (payload.res) {
```

Comment:
Same suggestion here about combining the conditions and early returns.

Reply:
and same answer